src/backend_sdl3_opengl.zig

```diff
@@ -0,0 +1,52 @@
+const gui = @import("gui.zig");
+const backend_sdl3 = @import("backend_sdl3.zig");
+
+pub fn initWithGlSlVersion(
+    window: *const anyopaque, // SDL_Window
+    context: *const anyopaque, // SDL_GL_Context
+    glsl_version: ?[:0]const u8, // e.g. "#version 130"
+) void {
+    backend_sdl3.initOpenGL(window, context);
+
+    ImGui_ImplOpenGL3_Init(@ptrCast(glsl_version));
+}
+
+pub fn init(
+    window: *const anyopaque, // SDL_Window
+    context: *const anyopaque, // SDL_GL_Context
+) void {
+    initWithGlSlVersion(window, context, null);
+}
+
+pub fn processEvent(
+    event: *const anyopaque, // SDL_Event
+) bool {
+    return backend_sdl3.processEvent(event);
+}
+
+pub fn deinit() void {
+    ImGui_ImplOpenGL3_Shutdown();
+    backend_sdl3.deinit();
+}
+
+pub fn newFrame(fb_width: u32, fb_height: u32) void {
+    ImGui_ImplOpenGL3_NewFrame();
+    backend_sdl3.newFrame();
+
+    gui.io.setDisplaySize(@as(f32, @floatFromInt(fb_width)), @as(f32, @floatFromInt(fb_height)));
+    gui.io.setDisplayFramebufferScale(1.0, 1.0);
```

Comment:
I think the order of things here is wrong. But we can come back to it. See the open PR re DPI scaling.

Reply:
I based this on the sdl2_opengl backend ^^. except that I do the opengl newframe before sdl3 newframe.
I seen that here: 
https://github.com/ocornut/imgui/blob/f484af34c2014e98a64410e6cd81e1a5ab434bfd/examples/example_sdl3_opengl3/main.cpp#L162C12-L165C1
But tbh I think the order has no importance as the init before that is in the reverse order.